GTG/gtk/ui/general_preferences.ui

```diff
@@ -0,0 +1,390 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <object class="GtkScrolledWindow" id="general_pref_window">
+    <property name="min-content-height">500</property>
```

Comment:
This should be set probably on the preferences window instead? (The window size shouldn't change when you switch between slides)